eksctl/gridSST.jsonnet

```
@@ -0,0 +1,109 @@
+// Exports an eksctl config file for carbonplan cluster
+local ng = import "./libsonnet/nodegroup.jsonnet";
+
+// place all cluster nodes here
+local clusterRegion = "us-west-2";
+local masterAzs = ["us-west-2a", "us-west-2b", "us-west-2c"];
+local nodeAz = "us-west-2a";
+
+// Node definitions for notebook nodes. Config here is merged
+// with our notebook node definition.
+// A `node.kubernetes.io/instance-type label is added, so pods
+// can request a particular kind of node with a nodeSelector
+local notebookNodes = [
+    { instanceType: "m5.large" },
+    { instanceType: "m5.xlarge" },
+    { instanceType: "m5.2xlarge" },
+    { instanceType: "m5.8xlarge" },
+];
+
+// Node definitions for dask worker nodes. Config here is merged
+// with our dask worker node definition, which uses spot instances.
+// A `node.kubernetes.io/instance-type label is set to the name of the
+// *first* item in instanceDistribution.instanceTypes, to match
+// what we do with notebook nodes. Pods can request a particular
+// kind of node with a nodeSelector
+local daskNodes = [
+    { instancesDistribution+: { instanceTypes: ["m5.large"] }},
+    { instancesDistribution+: { instanceTypes: ["m5.xlarge"] }},
+    { instancesDistribution+: { instanceTypes: ["m5.2xlarge"] }},
+    { instancesDistribution+: { instanceTypes: ["m5.8xlarge"] }},
+];
+
+{
+    apiVersion: 'eksctl.io/v1alpha5',
+    kind: 'ClusterConfig',
+    metadata+: {
+        name: "gridSST",
+        region: clusterRegion,
+        version: '1.22'
```

```suggestion
        version: '1.23'
```

Let's start off with 1.23 as that is latest available (https://docs.aws.amazon.com/eks/latest/userguide./kubernetes-versions.html)